### ISSUE TITLE
Убрать репозитории

### ISSUE THREAD

**Issue by jbmeerkat** [rocket-11/nastachku#12]
Определение паттерна [Repository](http://martinfowler.com/eaaCatalog/repository.html) и существующая реализация не имеют ничего общего. Чтобы не вносить путаницу и не расстраивать Дядюшку Боба необходимо перенести все скоупы обратно в модели.

**Comment by kalashnikovisme** [comment]
Так-то ваше дело. Ну вот прочитайте внимательно про репозитории на русском ;)
http://habrahabr.ru/post/248505/

**Comment by jbmeerkat** [comment]
Да, @kalashnikovisme прочитайте-ка вашу статью с хабра или, например, Фаулера или вообще любое другое определение, а потом внимательно-внимательно посмотрите на то, что из себя представляет текущая реализация в этом проекте.
Если, после всех этих стараний, понять почему текущая реализация не очень хороша все-таки не получится - я могу развернуто пояснить.
